netbox/extras/api/serializers_/tableconfigs.py

```diff
@@ -0,0 +1,22 @@
+from core.models import ObjectType
+from extras.models import TableConfig
+from netbox.api.fields import ContentTypeField
+from netbox.api.serializers import ValidatedModelSerializer
+
+__all__ = (
+    'TableConfigSerializer',
+)
+
+
+class TableConfigSerializer(ValidatedModelSerializer):
+    object_type = ContentTypeField(
+        queryset=ObjectType.objects.all()
+    )
+
+    class Meta:
+        model = TableConfig
+        fields = [
+            'id', 'url', 'display_url', 'display', 'object_type', 'table', 'name', 'slug', 'description', 'user',
+            'weight', 'enabled', 'shared', 'columns', 'ordering', 'created', 'last_updated',
+        ]
+        brief_fields = ('id', 'url', 'display', 'name', 'slug', 'description')
```

Comment:
Should there be some sort of indication of what model/table this applies to in `brief_fields`?

Reply:
`brief_fields` defines the set of fields returned by the serializer when `?brief=True` is passed with an API request. (It's a subset of the serializer's fields.)